src/style/markdown.less

```diff
@@ -381,9 +372,13 @@
 	h4,
 	h5,
 	h6 {
+		// fix anchor target positioning to account for fixed header
 		&:target {
 			padding-top: @header-and-banner-height;
 		}
+		content-region[name*='v8'] &:target {
+			padding-top: calc(@header-and-banner-height + 3.25rem);
```

Comment:
This is the only location in which the aggregate height for all 3 (banner, header, and v8 warning) matters, so I chose not to extract it out to a var.

Lmk if this is too magic number-y.